intro_to_ada.rst

```diff
@@ -1115,10 +1115,10 @@ end up containing a lot of conversions. However, this approach has some
 advantages. For example:
 
 - You can rely on the fact that no implicit conversion will ever happen in your
-  numeric code. In C for example, the rules for implicit conversions are very
-  non-obvious. In Ada the code will always do exactly what it seems to do.
-
-.. AI for gusthoff: Do something more explicit about that
+  numeric code. In C for example, the rules for implicit conversions may not
```

Comment:
We should add a LANGCOMP about the C comparison here I think, and maybe expand a bit inline.